TetraSwap.appdata.xml

```diff
@@ -0,0 +1,79 @@
+<!-- Copyright 2021 Eclipium -->
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+ <id>xyz.eclipium.TetraSwap</id>
+ <metadata_license>CC0-1.0</metadata_license>
+ <project_license>LicenseRef-proprietary</project_license>
+ <name>TetraSwap</name>
+ <icon type="remote" width="1024" height="1024">https://launcher.eclipium.xyz/Images/TetraSwap.png</icon>
+ <summary>A simple but challenging futuristic puzzle game!</summary>
+ <summary lang="fr_FR">Un jeu de réflexion futuriste simple mais difficile!</summary>
+ <categories>
+    <category>Game</category>
+ </categories>
+  <description>
+  <p>
+  TetraSwap is a futuristic puzzle game available on phone, tablet and computer.
+The game is composed of a board made of several colored squares (2 to 4 colors) as well as colored tokens and a gray token, the goal of the game being to place all the tokens on their corresponding colored squares, it could seem easy but think again! Because the only way to move them is to use the grey chip, which is complicated to master.
+There are 3 game modes:
+Classic: Possessing 48 levels (can increase with updates), with various game mechanics introduced over the levels.
+Random: The game randomly generates a board with a defined number of squares and colors.
+Daily: Take on daily challenges and try to beat your friends in the leaderboard.
+ </p>
+ </description>
+   <description lang="fr_FR">
+  <p>
+TetraSwap est un jeu de réflexion futuriste disponible sur téléphone tablette et ordinateur.
+Le jeu est composé d'un plateau composé de plusieurs cases colorées (2 à 4 couleurs) ainsi que de jetons colorés et d'un jeton gris, le but du jeu étant de placer tous les jetons sur leurs cases de couleur correspondante, cela pourrait vous sembler facile mais détrompez vous ! Car le seul moyen de les déplacer est d'utiliser le jeton gris, compliqué à maîtriser.
+Il y a 3 modes de jeu :
+Classique : Possédant 48 niveaux (peut augmenter au fil des mises à jour), avec diverses mécaniques de jeu introduites au fil des niveaux.
+Aléatoire : Le jeu génère aléatoirement un plateau avec un nombre de cases et de couleurs définis.
+Journalier : Relevez les défis journaliers et essayez de dépasser vos amis dans le classement.
+ </p>
```

Comment:
```suggestion
  <p>
TetraSwap est un jeu de réflexion futuriste disponible sur téléphone tablette et ordinateur.
</p>
<p>
Le jeu est composé d'un plateau composé de plusieurs cases colorées (2 à 4 couleurs) ainsi que de jetons colorés et d'un jeton gris, le but du jeu étant de placer tous les jetons sur leurs cases de couleur correspondante, cela pourrait vous sembler facile mais détrompez vous ! Car le seul moyen de les déplacer est d'utiliser le jeton gris, compliqué à maîtriser.
</p>
<p>
Il y a 3 modes de jeu :
</p>
<ul>
<li>Classique : Possédant 48 niveaux (peut augmenter au fil des mises à jour), avec diverses mécaniques de jeu introduites au fil des niveaux.</li>
<li>Aléatoire : Le jeu génère aléatoirement un plateau avec un nombre de cases et de couleurs définis.</li>
<li>Journalier : Relevez les défis journaliers et essayez de dépasser vos amis dans le classement.</li>
 </ul>
```

And likely same for the english version.
The problem is the version long single `<p>` that end up being displayed no the way you want.

Reply:
I corrected the problem with another way, thanks